lib/rpc.ts

```diff
@@ -135,8 +135,8 @@ export class Rpc extends EventEmitter implements IForwarderDest {
    */
   public start(sendMessage: SendMessageCB) {
     // Message sent by `_dispatch(...)` are appended to the send queue
+    processQueue(this._inactiveSendQueue, this._sendMessageOrReject.bind(this, sendMessage));
```

Comment:
I think the messages sent when dispatching the received messages don't get sent anymore, because when `_dispatch` is called `this._sendMessageCB` is `null`. This does not happen if we process `_inactiveSendQueue` after the `_inactiveRecvQueue`.